2015/01/07 18:54:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "page".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('page')
#1 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#2 /var/www/fs_yii/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/fs_yii/page?view=about
---
2015/01/07 19:47:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "estabelecimento".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#1 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#2 /var/www/fs_yii/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 19:47:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "estabelecimentos".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#1 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#2 /var/www/fs_yii/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/fs_yii/estabelecimentos
---
2015/01/07 19:48:46 [error] [exception.CException] exception 'CException' with message 'EstabelecimentosController cannot find the requested view "index".' in /var/www/yii/framework/web/CController.php:878
Stack trace:
#0 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('index', NULL, true)
#1 /var/www/fs_yii/protected/controllers/EstabelecimentosController.php(7): CController->render('index')
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentosController->actionIndex()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/estabelecimentos
---
2015/01/07 19:48:47 [error] [exception.CException] exception 'CException' with message 'EstabelecimentosController cannot find the requested view "index".' in /var/www/yii/framework/web/CController.php:878
Stack trace:
#0 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('index', NULL, true)
#1 /var/www/fs_yii/protected/controllers/EstabelecimentosController.php(7): CController->render('index')
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentosController->actionIndex()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/estabelecimentos
---
2015/01/07 19:49:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "estabelecimentos".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#1 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#2 /var/www/fs_yii/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/fs_yii/estabelecimentos
---
2015/01/07 19:54:12 [error] [exception.CDbException] exception 'CDbException' with message 'The table "Estabelecimentos" for active record class "Estabelecimento" cannot be found in the database.' in /var/www/yii/framework/db/ar/CActiveRecord.php:2385
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Estabelecimento))
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1355): CActiveRecord->getTableSchema()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(7): CActiveRecord->findAll()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 19:54:26 [error] [exception.CDbException] exception 'CDbException' with message 'The table "estabelecimentos" for active record class "Estabelecimento" cannot be found in the database.' in /var/www/yii/framework/db/ar/CActiveRecord.php:2385
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Estabelecimento))
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1355): CActiveRecord->getTableSchema()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(7): CActiveRecord->findAll()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 19:55:17 [error] [exception.CDbException] exception 'CDbException' with message 'The table "estabelecimentos" for active record class "Estabelecimento" cannot be found in the database.' in /var/www/yii/framework/db/ar/CActiveRecord.php:2385
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Estabelecimento))
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1355): CActiveRecord->getTableSchema()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(7): CActiveRecord->findAll()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 19:55:19 [error] [exception.CDbException] exception 'CDbException' with message 'The table "estabelecimentos" for active record class "Estabelecimento" cannot be found in the database.' in /var/www/yii/framework/db/ar/CActiveRecord.php:2385
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Estabelecimento))
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1355): CActiveRecord->getTableSchema()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(7): CActiveRecord->findAll()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 19:55:19 [error] [exception.CDbException] exception 'CDbException' with message 'The table "estabelecimentos" for active record class "Estabelecimento" cannot be found in the database.' in /var/www/yii/framework/db/ar/CActiveRecord.php:2385
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Estabelecimento))
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1355): CActiveRecord->getTableSchema()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(7): CActiveRecord->findAll()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 19:55:19 [error] [exception.CDbException] exception 'CDbException' with message 'The table "estabelecimentos" for active record class "Estabelecimento" cannot be found in the database.' in /var/www/yii/framework/db/ar/CActiveRecord.php:2385
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Estabelecimento))
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1355): CActiveRecord->getTableSchema()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(7): CActiveRecord->findAll()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 19:55:19 [error] [exception.CDbException] exception 'CDbException' with message 'The table "estabelecimentos" for active record class "Estabelecimento" cannot be found in the database.' in /var/www/yii/framework/db/ar/CActiveRecord.php:2385
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Estabelecimento))
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1355): CActiveRecord->getTableSchema()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(7): CActiveRecord->findAll()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 19:55:45 [error] [exception.CDbException] exception 'CDbException' with message 'The table "estabelecimentos" for active record class "Estabelecimento" cannot be found in the database.' in /var/www/yii/framework/db/ar/CActiveRecord.php:2385
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Estabelecimento))
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1355): CActiveRecord->getTableSchema()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(7): CActiveRecord->findAll()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 19:57:29 [error] [exception.CDbException] exception 'CDbException' with message 'The table "estabelecimentos" for active record class "Estabelecimento" cannot be found in the database.' in /var/www/yii/framework/db/ar/CActiveRecord.php:2385
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Estabelecimento))
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1355): CActiveRecord->getTableSchema()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(7): CActiveRecord->findAll()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 20:03:53 [error] [exception.CException] exception 'CException' with message 'Property "EstabelecimentoController.Estabelecimento" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(8): CComponent->__get('Estabelecimento')
#1 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#2 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#4 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#6 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#7 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#8 /var/www/fs_yii/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 20:04:40 [error] [exception.CException] exception 'CException' with message 'Property "EstabelecimentoController.Estabelecimento" is not defined.' in /var/www/yii/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(7): CComponent->__set('Estabelecimento', Object(Estabelecimento))
#1 /var/www/yii/framework/web/CWebApplication.php(359): EstabelecimentoController->__construct('estabelecimento', NULL)
#2 /var/www/yii/framework/web/CWebApplication.php(276): CWebApplication->createController('estabelecimento')
#3 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#4 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#5 /var/www/fs_yii/index.php(13): CApplication->run()
#6 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 20:05:08 [error] [exception.CException] exception 'CException' with message 'EstabelecimentoController cannot find the requested view "index".' in /var/www/yii/framework/web/CController.php:878
Stack trace:
#0 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('index', NULL, true)
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(18): CController->render('index')
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/estabelecimento
---
2015/01/07 20:42:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastrar".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('cadastrar')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastrar')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastrar
HTTP_REFERER=http://localhost/fs_yii/estabelecimento
---
2015/01/07 20:42:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastrar".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('cadastrar')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastrar')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastrar
HTTP_REFERER=http://localhost/fs_yii/estabelecimento
---
2015/01/07 20:42:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/estabelecimento
---
2015/01/07 20:43:15 [error] [php] Invalid argument supplied for foreach() (/var/www/fs_yii/protected/views/estabelecimento/index.php:10)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#1 /var/www/yii/framework/web/CController.php(782): EstabelecimentoController->renderPartial()
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(30): EstabelecimentoController->render()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#5 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#6 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#7 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#9 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#10 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
in /var/www/fs_yii/protected/views/estabelecimento/index.php (10)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (30)
in /var/www/fs_yii/index.php (13)
2015/01/07 20:44:50 [error] [exception.CException] exception 'CException' with message 'EstabelecimentoController contains improperly nested widget tags in its view "/var/www/fs_yii/protected/views/estabelecimento/cadastro.php". A CActiveForm widget does not have an endWidget() call.' in /var/www/yii/framework/web/CBaseController.php:101
Stack trace:
#0 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', NULL, true)
#1 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', NULL, true)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(30): CController->render('cadastro')
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#9 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#10 /var/www/fs_yii/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/estabelecimento
---
2015/01/07 20:46:05 [error] [exception.CException] exception 'CException' with message 'Property "EstabelecimentoController.Form" is not defined.' in /var/www/yii/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(1): CComponent->__set('Form', Object(CActiveForm))
#1 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#2 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', NULL, true)
#3 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', NULL, true)
#4 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', NULL, true)
#5 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(30): CController->render('cadastro')
#6 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#7 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#11 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#12 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#13 /var/www/fs_yii/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/estabelecimento
---
2015/01/07 20:49:21 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.name" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('name')
#1 /var/www/yii/framework/web/helpers/CHtml.php(2728): CActiveRecord->__get('name')
#2 /var/www/yii/framework/web/helpers/CHtml.php(2451): CHtml::resolveValue(Object(Estabelecimento), 'name')
#3 /var/www/yii/framework/web/helpers/CHtml.php(1533): CHtml::activeInputField('text', Object(Estabelecimento), 'name', Array)
#4 /var/www/yii/framework/web/widgets/CActiveForm.php(792): CHtml::activeTextField(Object(Estabelecimento), 'name', Array)
#5 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(4): CActiveForm->textField(Object(Estabelecimento), 'name')
#6 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#7 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#9 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#10 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(31): CController->render('cadastro', Array)
#11 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#12 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#13 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#14 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#16 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#17 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#18 /var/www/fs_yii/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/estabelecimento
---
2015/01/07 20:52:50 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.titulo" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('titulo')
#1 /var/www/yii/framework/web/helpers/CHtml.php(2728): CActiveRecord->__get('titulo')
#2 /var/www/yii/framework/web/helpers/CHtml.php(2451): CHtml::resolveValue(Object(Estabelecimento), 'titulo')
#3 /var/www/yii/framework/web/helpers/CHtml.php(1533): CHtml::activeInputField('text', Object(Estabelecimento), 'titulo', Array)
#4 /var/www/yii/framework/web/widgets/CActiveForm.php(792): CHtml::activeTextField(Object(Estabelecimento), 'titulo', Array)
#5 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(4): CActiveForm->textField(Object(Estabelecimento), 'titulo')
#6 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#7 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#9 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#10 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(31): CController->render('cadastro', Array)
#11 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#12 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#13 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#14 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#16 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#17 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#18 /var/www/fs_yii/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/estabelecimento
---
2015/01/07 20:52:52 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.titulo" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('titulo')
#1 /var/www/yii/framework/web/helpers/CHtml.php(2728): CActiveRecord->__get('titulo')
#2 /var/www/yii/framework/web/helpers/CHtml.php(2451): CHtml::resolveValue(Object(Estabelecimento), 'titulo')
#3 /var/www/yii/framework/web/helpers/CHtml.php(1533): CHtml::activeInputField('text', Object(Estabelecimento), 'titulo', Array)
#4 /var/www/yii/framework/web/widgets/CActiveForm.php(792): CHtml::activeTextField(Object(Estabelecimento), 'titulo', Array)
#5 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(4): CActiveForm->textField(Object(Estabelecimento), 'titulo')
#6 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#7 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#9 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#10 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(31): CController->render('cadastro', Array)
#11 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#12 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#13 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#14 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#16 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#17 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#18 /var/www/fs_yii/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/estabelecimento
---
2015/01/07 22:45:35 [error] [exception.CException] exception 'CException' with message 'Estabelecimento and its behaviors do not have a method or closure named "attributes".' in /var/www/yii/framework/base/CComponent.php:266
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(227): CComponent->__call('attributes', Array)
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(37): CActiveRecord->__call('attributes', Array)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(37): Estabelecimento->attributes(Array)
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#9 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#10 /var/www/fs_yii/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/07 23:16:01 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "nome" violates not-null constraint
DETAIL:  Failing row contains (15, null, null).. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("id") VALUES (DEFAULT).
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (46)
in /var/www/fs_yii/index.php (13)
2015/01/07 23:16:01 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "nome" violates not-null constraint
DETAIL:  Failing row contains (15, null, null).. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("id") VALUES (DEFAULT)' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(1080): CDbCommand->execute()
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(46): CActiveRecord->save()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#9 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#10 /var/www/fs_yii/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/index
---
2015/01/07 23:16:03 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "nome" violates not-null constraint
DETAIL:  Failing row contains (16, null, null).. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("id") VALUES (DEFAULT).
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (46)
in /var/www/fs_yii/index.php (13)
2015/01/07 23:16:03 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "nome" violates not-null constraint
DETAIL:  Failing row contains (16, null, null).. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("id") VALUES (DEFAULT)' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(1080): CDbCommand->execute()
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(46): CActiveRecord->save()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#9 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#10 /var/www/fs_yii/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/07 23:16:19 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "nome" violates not-null constraint
DETAIL:  Failing row contains (17, null, null).. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("id") VALUES (DEFAULT).
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (46)
in /var/www/fs_yii/index.php (13)
2015/01/07 23:16:19 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "nome" violates not-null constraint
DETAIL:  Failing row contains (17, null, null).. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("id") VALUES (DEFAULT)' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(1080): CDbCommand->execute()
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(46): CActiveRecord->save()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#9 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#10 /var/www/fs_yii/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/07 23:29:05 [warning] [application] Failed to set unsafe attribute "descricao" of "Estabelecimento".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (17)
in /var/www/fs_yii/index.php (13)
2015/01/07 23:29:05 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "descricao" violates not-null constraint
DETAIL:  Failing row contains (27, fgfdgdfg, null).. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("nome") VALUES (:yp0).
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (44)
in /var/www/fs_yii/index.php (13)
2015/01/07 23:29:05 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "descricao" violates not-null constraint
DETAIL:  Failing row contains (27, fgfdgdfg, null).. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("nome") VALUES (:yp0)' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(1080): CDbCommand->execute()
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(44): CActiveRecord->save()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#9 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#10 /var/www/fs_yii/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/07 23:29:17 [warning] [application] Failed to set unsafe attribute "descricao" of "Estabelecimento".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (17)
in /var/www/fs_yii/index.php (13)
2015/01/07 23:29:17 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "descricao" violates not-null constraint
DETAIL:  Failing row contains (28, fdfdsfdsf, null).. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("nome") VALUES (:yp0).
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (44)
in /var/www/fs_yii/index.php (13)
2015/01/07 23:29:17 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "descricao" violates not-null constraint
DETAIL:  Failing row contains (28, fdfdsfdsf, null).. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("nome") VALUES (:yp0)' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(1080): CDbCommand->execute()
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(44): CActiveRecord->save()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#9 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#10 /var/www/fs_yii/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/07 23:44:59 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /var/www/yii/framework/web/CController.php:336
Stack trace:
#0 /var/www/yii/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#4 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#5 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#6 /var/www/fs_yii/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar
---
2015/01/08 12:18:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "demos/blog/index-test.php/contact/1".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('demos/blog/inde...')
#1 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#2 /var/www/fs_yii/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/fs_yii/demos/blog/index-test.php/contact/1
---
2015/01/08 12:18:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "demos/blog/index-test.php/contact".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('demos/blog/inde...')
#1 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#2 /var/www/fs_yii/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/fs_yii/demos/blog/index-test.php/contact/
---
2015/01/08 12:18:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "demos/blog/index-test.php/site".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('demos/blog/inde...')
#1 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#2 /var/www/fs_yii/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/fs_yii/demos/blog/index-test.php/site/
---
2015/01/09 09:24:10 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /var/www/yii/framework/web/CController.php:336
Stack trace:
#0 /var/www/yii/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#4 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#5 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#6 /var/www/fs_yii/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/
---
2015/01/09 09:24:33 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /var/www/yii/framework/web/CController.php:336
Stack trace:
#0 /var/www/yii/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#4 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#5 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#6 /var/www/fs_yii/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/
---
2015/01/09 09:24:34 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /var/www/yii/framework/web/CController.php:336
Stack trace:
#0 /var/www/yii/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#4 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#5 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#6 /var/www/fs_yii/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/
---
2015/01/09 09:25:12 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /var/www/yii/framework/web/CController.php:336
Stack trace:
#0 /var/www/yii/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#4 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#5 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#6 /var/www/fs_yii/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/
---
2015/01/09 09:25:14 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /var/www/yii/framework/web/CController.php:336
Stack trace:
#0 /var/www/yii/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#4 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#5 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#6 /var/www/fs_yii/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/
---
2015/01/09 09:25:15 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /var/www/yii/framework/web/CController.php:336
Stack trace:
#0 /var/www/yii/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#4 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#5 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#6 /var/www/fs_yii/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/
---
2015/01/09 09:25:15 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /var/www/yii/framework/web/CController.php:336
Stack trace:
#0 /var/www/yii/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#4 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#5 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#6 /var/www/fs_yii/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/
---
2015/01/09 09:25:28 [error] [exception.CException] exception 'CException' with message 'EstabelecimentoController and its behaviors do not have a method or closure named "loadModel".' in /var/www/yii/framework/base/CComponent.php:266
Stack trace:
#0 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(52): CComponent->__call('loadModel', Array)
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(52): EstabelecimentoController->loadModel()
#2 [internal function]: EstabelecimentoController->actionEditar('1')
#3 /var/www/yii/framework/web/actions/CAction.php(109): ReflectionMethod->invokeArgs(Object(EstabelecimentoController), Array)
#4 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(EstabelecimentoController), Object(ReflectionMethod), Array)
#5 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#9 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#10 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#11 /var/www/fs_yii/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/1
---
2015/01/09 09:26:14 [error] [php] Argument 1 passed to EstabelecimentoController::actionEditar() must be an instance of int, string given (/var/www/fs_yii/protected/controllers/EstabelecimentoController.php:51)
Stack trace:
#0 /var/www/yii/framework/web/actions/CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#2 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#3 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#4 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#5 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#6 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#7 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/1
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (51)
in /var/www/fs_yii/index.php (13)
2015/01/09 09:26:19 [error] [php] Argument 1 passed to EstabelecimentoController::actionEditar() must be an instance of int, string given (/var/www/fs_yii/protected/controllers/EstabelecimentoController.php:51)
Stack trace:
#0 /var/www/yii/framework/web/actions/CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#2 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#3 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#4 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#5 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#6 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#7 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/1
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (51)
in /var/www/fs_yii/index.php (13)
2015/01/09 09:49:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php:41
Stack trace:
#0 [internal function]: EstabelecimentoController->actionEditar('41')
#1 /var/www/yii/framework/web/actions/CAction.php(109): ReflectionMethod->invokeArgs(Object(EstabelecimentoController), Array)
#2 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(EstabelecimentoController), Object(ReflectionMethod), Array)
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/41
---
2015/01/09 09:57:37 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimentos_pkey"
DETAIL:  Key (id)=(43) already exists.. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("nome", "descricao", "id") VALUES (:yp0, :yp1, :yp2).
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (54)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (46)
in /var/www/fs_yii/index.php (13)
2015/01/09 09:57:37 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimentos_pkey"
DETAIL:  Key (id)=(43) already exists.. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("nome", "descricao", "id") VALUES (:yp0, :yp1, :yp2)' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(1080): CDbCommand->execute()
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(54): CActiveRecord->save()
#3 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(46): EstabelecimentoController->saveOrUpdate()
#4 [internal function]: EstabelecimentoController->actionEditar('43')
#5 /var/www/yii/framework/web/actions/CAction.php(109): ReflectionMethod->invokeArgs(Object(EstabelecimentoController), Array)
#6 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(EstabelecimentoController), Object(ReflectionMethod), Array)
#7 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#11 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#12 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#13 /var/www/fs_yii/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/43
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/editar/43
---
2015/01/09 09:57:45 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimentos_pkey"
DETAIL:  Key (id)=(43) already exists.. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("nome", "descricao", "id") VALUES (:yp0, :yp1, :yp2).
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (54)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (46)
in /var/www/fs_yii/index.php (13)
2015/01/09 09:57:45 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimentos_pkey"
DETAIL:  Key (id)=(43) already exists.. The SQL statement executed was: INSERT INTO "yii_estabelecimentos" ("nome", "descricao", "id") VALUES (:yp0, :yp1, :yp2)' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(1080): CDbCommand->execute()
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(54): CActiveRecord->save()
#3 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(46): EstabelecimentoController->saveOrUpdate()
#4 [internal function]: EstabelecimentoController->actionEditar('43')
#5 /var/www/yii/framework/web/actions/CAction.php(109): ReflectionMethod->invokeArgs(Object(EstabelecimentoController), Array)
#6 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(EstabelecimentoController), Object(ReflectionMethod), Array)
#7 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#11 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#12 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#13 /var/www/fs_yii/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/43
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/editar/43
---
2015/01/09 09:58:06 [error] [exception.CDbException] exception 'CDbException' with message 'The active record cannot be updated because it is new.' in /var/www/yii/framework/db/ar/CActiveRecord.php:1121
Stack trace:
#0 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(54): CActiveRecord->update()
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(46): EstabelecimentoController->saveOrUpdate()
#2 [internal function]: EstabelecimentoController->actionEditar('43')
#3 /var/www/yii/framework/web/actions/CAction.php(109): ReflectionMethod->invokeArgs(Object(EstabelecimentoController), Array)
#4 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(EstabelecimentoController), Object(ReflectionMethod), Array)
#5 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#9 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#10 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#11 /var/www/fs_yii/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/43
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/editar/43
---
2015/01/09 10:05:08 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento._new" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('_new')
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(42): CActiveRecord->__get('_new')
#2 [internal function]: EstabelecimentoController->actionEditar('43')
#3 /var/www/yii/framework/web/actions/CAction.php(109): ReflectionMethod->invokeArgs(Object(EstabelecimentoController), Array)
#4 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(EstabelecimentoController), Object(ReflectionMethod), Array)
#5 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#9 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#10 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#11 /var/www/fs_yii/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/43
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/editar/43
---
2015/01/09 10:13:59 [error] [php] Invalid argument supplied for foreach() (/var/www/yii/framework/web/CUrlManager.php:286)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(982): EstabelecimentoController->createUrl()
#1 /var/www/fs_yii/protected/views/estabelecimento/index.php(32): EstabelecimentoController->createAbsoluteUrl()
#2 /var/www/yii/framework/web/CBaseController.php(126): require()
#3 /var/www/yii/framework/web/CBaseController.php(95): EstabelecimentoController->renderInternal()
#4 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#5 /var/www/yii/framework/web/CController.php(782): EstabelecimentoController->renderPartial()
#6 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(21): EstabelecimentoController->render()
#7 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#8 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#9 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#10 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#11 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#12 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#13 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#14 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/index
in /var/www/fs_yii/protected/views/estabelecimento/index.php (32)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (21)
in /var/www/fs_yii/index.php (13)
2015/01/09 13:17:23 [error] [exception.CException] exception 'CException' with message 'EstabelecimentoController cannot find the requested view "_address".' in /var/www/yii/framework/web/CController.php:878
Stack trace:
#0 /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php(18): CController->renderPartial('_address', Array)
#1 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#2 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#3 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#4 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): CController->renderPartial('_form_save_or_u...', Array)
#5 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#6 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#7 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#9 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(33): CController->render('cadastro', Array)
#10 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#11 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#12 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#13 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#15 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#16 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#17 /var/www/fs_yii/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/index
---
2015/01/09 13:22:48 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.logradouro" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('logradouro')
#1 /var/www/yii/framework/web/helpers/CHtml.php(2728): CActiveRecord->__get('logradouro')
#2 /var/www/yii/framework/web/helpers/CHtml.php(2451): CHtml::resolveValue(Object(Estabelecimento), 'logradouro')
#3 /var/www/yii/framework/web/helpers/CHtml.php(1533): CHtml::activeInputField('text', Object(Estabelecimento), 'logradouro', Array)
#4 /var/www/yii/framework/web/widgets/CActiveForm.php(792): CHtml::activeTextField(Object(Estabelecimento), 'logradouro', Array)
#5 /var/www/fs_yii/protected/views/estabelecimento/_address.php(3): CActiveForm->textField(Object(Estabelecimento), 'logradouro')
#6 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#7 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#9 /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php(18): CController->renderPartial('_address', Array)
#10 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#11 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#12 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#13 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): CController->renderPartial('_form_save_or_u...', Array)
#14 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#15 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#16 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#17 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#18 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(33): CController->render('cadastro', Array)
#19 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#20 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#21 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#22 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#24 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#25 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#26 /var/www/fs_yii/index.php(13): CApplication->run()
#27 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/index
---
2015/01/09 13:22:49 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.logradouro" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('logradouro')
#1 /var/www/yii/framework/web/helpers/CHtml.php(2728): CActiveRecord->__get('logradouro')
#2 /var/www/yii/framework/web/helpers/CHtml.php(2451): CHtml::resolveValue(Object(Estabelecimento), 'logradouro')
#3 /var/www/yii/framework/web/helpers/CHtml.php(1533): CHtml::activeInputField('text', Object(Estabelecimento), 'logradouro', Array)
#4 /var/www/yii/framework/web/widgets/CActiveForm.php(792): CHtml::activeTextField(Object(Estabelecimento), 'logradouro', Array)
#5 /var/www/fs_yii/protected/views/estabelecimento/_address.php(3): CActiveForm->textField(Object(Estabelecimento), 'logradouro')
#6 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#7 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#9 /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php(18): CController->renderPartial('_address', Array)
#10 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#11 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#12 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#13 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): CController->renderPartial('_form_save_or_u...', Array)
#14 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#15 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#16 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#17 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#18 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(33): CController->render('cadastro', Array)
#19 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#20 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#21 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#22 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#24 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#25 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#26 /var/www/fs_yii/index.php(13): CApplication->run()
#27 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/index
---
2015/01/09 13:22:49 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.logradouro" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('logradouro')
#1 /var/www/yii/framework/web/helpers/CHtml.php(2728): CActiveRecord->__get('logradouro')
#2 /var/www/yii/framework/web/helpers/CHtml.php(2451): CHtml::resolveValue(Object(Estabelecimento), 'logradouro')
#3 /var/www/yii/framework/web/helpers/CHtml.php(1533): CHtml::activeInputField('text', Object(Estabelecimento), 'logradouro', Array)
#4 /var/www/yii/framework/web/widgets/CActiveForm.php(792): CHtml::activeTextField(Object(Estabelecimento), 'logradouro', Array)
#5 /var/www/fs_yii/protected/views/estabelecimento/_address.php(3): CActiveForm->textField(Object(Estabelecimento), 'logradouro')
#6 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#7 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#9 /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php(18): CController->renderPartial('_address', Array)
#10 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#11 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#12 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#13 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): CController->renderPartial('_form_save_or_u...', Array)
#14 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#15 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#16 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#17 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#18 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(33): CController->render('cadastro', Array)
#19 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#20 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#21 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#22 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#24 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#25 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#26 /var/www/fs_yii/index.php(13): CApplication->run()
#27 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/index
---
2015/01/09 13:22:50 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.logradouro" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('logradouro')
#1 /var/www/yii/framework/web/helpers/CHtml.php(2728): CActiveRecord->__get('logradouro')
#2 /var/www/yii/framework/web/helpers/CHtml.php(2451): CHtml::resolveValue(Object(Estabelecimento), 'logradouro')
#3 /var/www/yii/framework/web/helpers/CHtml.php(1533): CHtml::activeInputField('text', Object(Estabelecimento), 'logradouro', Array)
#4 /var/www/yii/framework/web/widgets/CActiveForm.php(792): CHtml::activeTextField(Object(Estabelecimento), 'logradouro', Array)
#5 /var/www/fs_yii/protected/views/estabelecimento/_address.php(3): CActiveForm->textField(Object(Estabelecimento), 'logradouro')
#6 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#7 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#9 /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php(18): CController->renderPartial('_address', Array)
#10 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#11 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#12 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#13 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): CController->renderPartial('_form_save_or_u...', Array)
#14 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#15 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#16 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#17 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#18 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(33): CController->render('cadastro', Array)
#19 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#20 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#21 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#22 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#24 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#25 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#26 /var/www/fs_yii/index.php(13): CApplication->run()
#27 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/index
---
2015/01/09 13:22:50 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.logradouro" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('logradouro')
#1 /var/www/yii/framework/web/helpers/CHtml.php(2728): CActiveRecord->__get('logradouro')
#2 /var/www/yii/framework/web/helpers/CHtml.php(2451): CHtml::resolveValue(Object(Estabelecimento), 'logradouro')
#3 /var/www/yii/framework/web/helpers/CHtml.php(1533): CHtml::activeInputField('text', Object(Estabelecimento), 'logradouro', Array)
#4 /var/www/yii/framework/web/widgets/CActiveForm.php(792): CHtml::activeTextField(Object(Estabelecimento), 'logradouro', Array)
#5 /var/www/fs_yii/protected/views/estabelecimento/_address.php(3): CActiveForm->textField(Object(Estabelecimento), 'logradouro')
#6 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#7 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#9 /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php(18): CController->renderPartial('_address', Array)
#10 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#11 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#12 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#13 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): CController->renderPartial('_form_save_or_u...', Array)
#14 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#15 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#16 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#17 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#18 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(33): CController->render('cadastro', Array)
#19 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#20 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#21 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#22 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#24 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#25 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#26 /var/www/fs_yii/index.php(13): CApplication->run()
#27 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/index
---
2015/01/09 13:29:34 [error] [php] include(Endereco.php): failed to open stream: No such file or directory (/var/www/yii/framework/YiiBase.php:432)
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(395): spl_autoload_call()
#1 /var/www/yii/framework/db/ar/CActiveFinder.php(182): model()
#2 /var/www/yii/framework/db/ar/CActiveFinder.php(230): CActiveFinder->getModel()
#3 /var/www/yii/framework/db/ar/CActiveFinder.php(48): CActiveFinder->buildJoinTree()
#4 /var/www/yii/framework/db/ar/CActiveRecord.php(912): CActiveFinder->__construct()
#5 /var/www/yii/framework/db/ar/CActiveRecord.php(280): Estabelecimento->getActiveFinder()
#6 /var/www/yii/framework/db/ar/CActiveRecord.php(143): Estabelecimento->getRelated()
#7 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(20): Estabelecimento->__get()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#10 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#11 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#12 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#14 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/index
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (20)
in /var/www/fs_yii/index.php (13)
2015/01/09 13:29:36 [error] [php] include(Endereco.php): failed to open stream: No such file or directory (/var/www/yii/framework/YiiBase.php:432)
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(395): spl_autoload_call()
#1 /var/www/yii/framework/db/ar/CActiveFinder.php(182): model()
#2 /var/www/yii/framework/db/ar/CActiveFinder.php(230): CActiveFinder->getModel()
#3 /var/www/yii/framework/db/ar/CActiveFinder.php(48): CActiveFinder->buildJoinTree()
#4 /var/www/yii/framework/db/ar/CActiveRecord.php(912): CActiveFinder->__construct()
#5 /var/www/yii/framework/db/ar/CActiveRecord.php(280): Estabelecimento->getActiveFinder()
#6 /var/www/yii/framework/db/ar/CActiveRecord.php(143): Estabelecimento->getRelated()
#7 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(20): Estabelecimento->__get()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#10 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#11 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#12 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#14 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/index
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (20)
in /var/www/fs_yii/index.php (13)
2015/01/09 13:31:29 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{endereco}}" for active record class "Endereco" cannot be found in the database.' in /var/www/yii/framework/db/ar/CActiveRecord.php:2385
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Endereco))
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 /var/www/yii/framework/db/ar/CActiveFinder.php(386): CActiveRecord->getTableSchema()
#3 /var/www/yii/framework/db/ar/CActiveFinder.php(281): CJoinElement->__construct(Object(CActiveFinder), Object(CManyManyRelation), Object(CJoinElement), 1)
#4 /var/www/yii/framework/db/ar/CActiveFinder.php(48): CActiveFinder->buildJoinTree(Object(CJoinElement), 'enderecos')
#5 /var/www/yii/framework/db/ar/CActiveRecord.php(912): CActiveFinder->__construct(Object(Estabelecimento), 'enderecos')
#6 /var/www/yii/framework/db/ar/CActiveRecord.php(280): CActiveRecord->getActiveFinder('enderecos')
#7 /var/www/yii/framework/db/ar/CActiveRecord.php(143): CActiveRecord->getRelated('enderecos')
#8 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(20): CActiveRecord->__get('enderecos')
#9 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#10 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#12 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#14 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#15 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#16 /var/www/fs_yii/index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/index
---
2015/01/09 13:31:40 [error] [exception.CDbException] exception 'CDbException' with message 'The relation "enderecos" in active record class "Estabelecimento" is not specified correctly: the join table "estabelecimento_enderecos" given in the foreign key cannot be found in the database.' in /var/www/yii/framework/db/ar/CActiveFinder.php:547
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveFinder.php(502): CJoinElement->applyLazyCondition(Object(CJoinQuery), Object(Estabelecimento))
#1 /var/www/yii/framework/db/ar/CActiveFinder.php(163): CJoinElement->lazyFind(Object(Estabelecimento))
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(281): CActiveFinder->lazyFind(Object(Estabelecimento))
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(143): CActiveRecord->getRelated('enderecos')
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(20): CActiveRecord->__get('enderecos')
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionIndex()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/index
---
2015/01/09 18:47:35 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.endereco" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('endereco')
#1 /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php(19): CActiveRecord->__get('endereco')
#2 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#3 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#4 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#5 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): CController->renderPartial('_form_save_or_u...', Array)
#6 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#7 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#9 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#10 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(33): CController->render('cadastro', Array)
#11 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#12 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#13 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#14 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#16 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#17 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#18 /var/www/fs_yii/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/09 18:59:01 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.endereco" is not defined.' in /var/www/yii/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(161): CComponent->__set('endereco', Object(Endereco))
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(27): CActiveRecord->__set('endereco', Object(Endereco))
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/09 18:59:28 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.endereco" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('endereco')
#1 /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php(19): CActiveRecord->__get('endereco')
#2 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#3 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#4 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#5 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): CController->renderPartial('_form_save_or_u...', Array)
#6 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#7 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#9 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#10 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(37): CController->render('cadastro', Array)
#11 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#12 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#13 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#14 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#16 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#17 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#18 /var/www/fs_yii/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 11:39:54 [error] [exception.CException] exception 'CException' with message 'Property "Endereco.{n}.cep" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('{n}.cep')
#1 /var/www/yii/framework/web/helpers/CHtml.php(2728): CActiveRecord->__get('{n}.cep')
#2 /var/www/yii/framework/web/helpers/CHtml.php(2451): CHtml::resolveValue(Object(Endereco), '{n}.cep')
#3 /var/www/yii/framework/web/helpers/CHtml.php(1533): CHtml::activeInputField('text', Object(Endereco), '{n}.cep', Array)
#4 /var/www/yii/framework/web/widgets/CActiveForm.php(792): CHtml::activeTextField(Object(Endereco), '{n}.cep', Array)
#5 /var/www/fs_yii/protected/views/estabelecimento/_address.php(33): CActiveForm->textField(Object(Endereco), '{n}.cep')
#6 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#7 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#8 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#9 /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php(19): CController->renderPartial('_address', Array)
#10 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#11 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#12 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#13 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): CController->renderPartial('_form_save_or_u...', Array)
#14 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/fs_yii...')
#15 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/fs_yii...', Array, true)
#16 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/fs_yii...', Array, true)
#17 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('cadastro', Array, true)
#18 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(41): CController->render('cadastro', Array)
#19 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#20 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#21 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#22 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#24 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#25 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#26 /var/www/fs_yii/index.php(13): CApplication->run()
#27 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 11:50:12 [warning] [application] Failed to set unsafe attribute "0" of "Endereco".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (33)
in /var/www/fs_yii/index.php (13)
2015/01/10 11:50:12 [warning] [application] Failed to set unsafe attribute "1" of "Endereco".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (33)
in /var/www/fs_yii/index.php (13)
2015/01/10 11:51:02 [warning] [application] Failed to set unsafe attribute "0" of "Endereco".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (33)
in /var/www/fs_yii/index.php (13)
2015/01/10 11:51:02 [warning] [application] Failed to set unsafe attribute "1" of "Endereco".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (33)
in /var/www/fs_yii/index.php (13)
2015/01/10 11:51:53 [warning] [application] Failed to set unsafe attribute "0" of "Endereco".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (33)
in /var/www/fs_yii/index.php (13)
2015/01/10 11:51:53 [warning] [application] Failed to set unsafe attribute "1" of "Endereco".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (33)
in /var/www/fs_yii/index.php (13)
2015/01/10 12:09:55 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.endereco" is not defined.' in /var/www/yii/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(161): CComponent->__set('endereco', Array)
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(33): CActiveRecord->__set('endereco', Array)
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 12:18:02 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.estabelecimento_enderecos" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('estabelecimento...')
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(27): CActiveRecord->__get('estabelecimento...')
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 12:18:28 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.aEnderecos" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('aEnderecos')
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(37): CActiveRecord->__get('aEnderecos')
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 12:18:54 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.aEnderecos" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('aEnderecos')
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(37): CActiveRecord->__get('aEnderecos')
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 12:37:02 [error] [exception.CException] exception 'CException' with message 'Property "CDbConnection.estabelecimentos" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(27): CComponent->__get('estabelecimento...')
#1 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#2 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#4 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#6 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#7 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#8 /var/www/fs_yii/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 12:38:10 [error] [exception.CException] exception 'CException' with message 'Property "CDbConnection._tables" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(27): CComponent->__get('_tables')
#1 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#2 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#4 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#6 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#7 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#8 /var/www/fs_yii/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 12:38:35 [error] [exception.CException] exception 'CException' with message 'Property "CDbConnection._schema" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(27): CComponent->__get('_schema')
#1 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#2 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#4 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#6 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#7 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#8 /var/www/fs_yii/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 12:41:26 [error] [exception.CException] exception 'CException' with message 'Property "CDbConnection._driverName" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(27): CComponent->__get('_driverName')
#1 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#2 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#4 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#6 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#7 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#8 /var/www/fs_yii/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 12:41:42 [error] [exception.CException] exception 'CException' with message 'Property "CDbConnection.CdbConnection" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(27): CComponent->__get('CdbConnection')
#1 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#2 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#4 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#6 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#7 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#8 /var/www/fs_yii/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 12:42:06 [error] [exception.CException] exception 'CException' with message 'Property "CDbConnection.columnTypes" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(27): CComponent->__get('columnTypes')
#1 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#2 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#4 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#6 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#7 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#8 /var/www/fs_yii/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 12:42:32 [error] [exception.CException] exception 'CException' with message 'Property "CDbConnection.CPgsqlSchema" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(27): CComponent->__get('CPgsqlSchema')
#1 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#2 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#4 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#6 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#7 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#8 /var/www/fs_yii/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 13:08:57 [error] [exception.CException] exception 'CException' with message 'Property "Estabelecimento.relations" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('relations')
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(102): CActiveRecord->__get('relations')
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1100): Estabelecimento->afterSave()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(78): CActiveRecord->save()
#5 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(42): EstabelecimentoController->saveOrUpdate()
#6 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#7 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#11 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#12 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#13 /var/www/fs_yii/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 13:09:19 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimento_enderecos_endereco_id_key"
DETAIL:  Key (endereco_id)=(2) already exists.. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('85', '2').
in /var/www/fs_yii/protected/models/Estabelecimento.php (116)
in /var/www/fs_yii/protected/models/Estabelecimento.php (103)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (78)
2015/01/10 13:09:19 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimento_enderecos_endereco_id_key"
DETAIL:  Key (endereco_id)=(2) already exists.. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('85', '2')' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(116): CDbCommand->execute()
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(103): Estabelecimento->insertManyManyEntry('insert into {{e...')
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1100): Estabelecimento->afterSave()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(78): CActiveRecord->save()
#5 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(42): EstabelecimentoController->saveOrUpdate()
#6 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#7 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#11 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#12 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#13 /var/www/fs_yii/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 13:11:21 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimento_enderecos_endereco_id_key"
DETAIL:  Key (endereco_id)=(3) already exists.. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('86', '3').
in /var/www/fs_yii/protected/models/Estabelecimento.php (116)
in /var/www/fs_yii/protected/models/Estabelecimento.php (103)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (78)
2015/01/10 13:11:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimento_enderecos_endereco_id_key"
DETAIL:  Key (endereco_id)=(3) already exists.. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('86', '3')' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(116): CDbCommand->execute()
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(103): Estabelecimento->insertManyManyEntry('insert into {{e...')
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1100): Estabelecimento->afterSave()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(78): CActiveRecord->save()
#5 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(42): EstabelecimentoController->saveOrUpdate()
#6 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#7 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#11 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#12 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#13 /var/www/fs_yii/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 13:12:12 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimento_enderecos_endereco_id_key"
DETAIL:  Key (endereco_id)=(4) already exists.. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('87', '4').
in /var/www/fs_yii/protected/models/Estabelecimento.php (116)
in /var/www/fs_yii/protected/models/Estabelecimento.php (103)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (78)
2015/01/10 13:12:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimento_enderecos_endereco_id_key"
DETAIL:  Key (endereco_id)=(4) already exists.. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('87', '4')' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(116): CDbCommand->execute()
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(103): Estabelecimento->insertManyManyEntry('insert into {{e...')
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1100): Estabelecimento->afterSave()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(78): CActiveRecord->save()
#5 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(42): EstabelecimentoController->saveOrUpdate()
#6 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#7 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#11 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#12 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#13 /var/www/fs_yii/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 13:13:56 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimento_enderecos_endereco_id_key"
DETAIL:  Key (endereco_id)=(5) already exists.. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('88', '5').
in /var/www/fs_yii/protected/models/Estabelecimento.php (116)
in /var/www/fs_yii/protected/models/Estabelecimento.php (103)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (78)
2015/01/10 13:13:56 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "yii_estabelecimento_enderecos_endereco_id_key"
DETAIL:  Key (endereco_id)=(5) already exists.. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('88', '5')' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(116): CDbCommand->execute()
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(103): Estabelecimento->insertManyManyEntry('insert into {{e...')
#2 /var/www/yii/framework/db/ar/CActiveRecord.php(1100): Estabelecimento->afterSave()
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(78): CActiveRecord->save()
#5 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(42): EstabelecimentoController->saveOrUpdate()
#6 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#7 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#11 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#12 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#13 /var/www/fs_yii/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 13:39:30 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "'95'"
LINE 1: insert into  values ('95', '11')
                             ^. The SQL statement executed was: insert into  values ('95', '11').
in /var/www/fs_yii/protected/models/Estabelecimento.php (127)
in /var/www/fs_yii/protected/models/Estabelecimento.php (115)
in /var/www/fs_yii/protected/models/Estabelecimento.php (100)
2015/01/10 13:39:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "'95'"
LINE 1: insert into  values ('95', '11')
                             ^. The SQL statement executed was: insert into  values ('95', '11')' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(127): CDbCommand->execute()
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(115): Estabelecimento->insertManyManyEntry('insert into  va...')
#2 /var/www/fs_yii/protected/models/Estabelecimento.php(100): Estabelecimento->saveRelations('enderecos', Array, Array)
#3 /var/www/yii/framework/db/ar/CActiveRecord.php(1100): Estabelecimento->afterSave()
#4 /var/www/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#5 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(78): CActiveRecord->save()
#6 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(42): EstabelecimentoController->saveOrUpdate()
#7 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#8 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#10 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#12 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#13 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#14 /var/www/fs_yii/index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 14:37:37 [error] [exception.CException] exception 'CException' with message 'an Error occured while trying saveRelations' in /var/www/fs_yii/protected/models/Estabelecimento.php:125
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(109): Estabelecimento->saveRelations('enderecos', Array, Array)
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(78): Estabelecimento->saveManyToMany()
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(42): EstabelecimentoController->saveOrUpdate()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#9 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#10 /var/www/fs_yii/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 14:38:14 [error] [exception.CException] exception 'CException' with message 'an Error occured while trying saveRelations' in /var/www/fs_yii/protected/models/Estabelecimento.php:125
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(109): Estabelecimento->saveRelations('enderecos', Array, Array)
#1 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(78): Estabelecimento->saveManyToMany()
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(42): EstabelecimentoController->saveOrUpdate()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#9 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#10 /var/www/fs_yii/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 14:47:45 [warning] [application] Failed to set unsafe attribute "0" of "Estabelecimento".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (44)
in /var/www/fs_yii/index.php (13)
2015/01/10 14:47:45 [warning] [application] Failed to set unsafe attribute "1" of "Estabelecimento".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (44)
in /var/www/fs_yii/index.php (13)
2015/01/10 14:48:20 [warning] [application] Failed to set unsafe attribute "0" of "Endereco".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (44)
in /var/www/fs_yii/index.php (13)
2015/01/10 14:48:20 [warning] [application] Failed to set unsafe attribute "1" of "Endereco".
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (44)
in /var/www/fs_yii/index.php (13)
2015/01/10 15:07:18 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for integer: ""
LINE 1: ...enderecos(estabelecimento_id, endereco_id) values ('', '35')
                                                              ^. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('', '35').
in /var/www/fs_yii/protected/models/Estabelecimento.php (132)
in /var/www/fs_yii/protected/models/Estabelecimento.php (124)
in /var/www/fs_yii/protected/models/Estabelecimento.php (107)
2015/01/10 15:07:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for integer: ""
LINE 1: ...enderecos(estabelecimento_id, endereco_id) values ('', '35')
                                                              ^. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('', '35')' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(132): CDbCommand->execute()
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(124): Estabelecimento->saveAssociativeTable('{{estabelecimen...', NULL, '35')
#2 /var/www/fs_yii/protected/models/Estabelecimento.php(107): Estabelecimento->saveRelations('enderecos', Array, Array)
#3 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(82): Estabelecimento->saveManyToMany()
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(39): EstabelecimentoController->saveOrUpdate()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 15:08:25 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for integer: ""
LINE 1: ...enderecos(estabelecimento_id, endereco_id) values ('', '36')
                                                              ^. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('', '36').
in /var/www/fs_yii/protected/models/Estabelecimento.php (132)
in /var/www/fs_yii/protected/models/Estabelecimento.php (124)
in /var/www/fs_yii/protected/models/Estabelecimento.php (107)
2015/01/10 15:08:25 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for integer: ""
LINE 1: ...enderecos(estabelecimento_id, endereco_id) values ('', '36')
                                                              ^. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('', '36')' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(132): CDbCommand->execute()
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(124): Estabelecimento->saveAssociativeTable('{{estabelecimen...', NULL, '36')
#2 /var/www/fs_yii/protected/models/Estabelecimento.php(107): Estabelecimento->saveRelations('enderecos', Array, Array)
#3 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(82): Estabelecimento->saveManyToMany()
#4 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(39): EstabelecimentoController->saveOrUpdate()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 15:46:26 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for integer: ""
LINE 1: ...enderecos(estabelecimento_id, endereco_id) values ('', '58')
                                                              ^. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('', '58').
in /var/www/fs_yii/protected/models/Estabelecimento.php (152)
in /var/www/fs_yii/protected/models/Estabelecimento.php (120)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (79)
2015/01/10 15:46:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for integer: ""
LINE 1: ...enderecos(estabelecimento_id, endereco_id) values ('', '58')
                                                              ^. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('', '58')' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(152): CDbCommand->execute()
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(120): Estabelecimento->saveAssociativeTable('{{estabelecimen...', NULL, '58')
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(79): Estabelecimento->saveManyToMany()
#3 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(42): EstabelecimentoController->saveOrUpdate()
#4 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#5 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#9 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#10 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#11 /var/www/fs_yii/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 15:47:28 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for integer: ""
LINE 1: ...enderecos(estabelecimento_id, endereco_id) values ('', '59')
                                                              ^. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('', '59').
in /var/www/fs_yii/protected/models/Estabelecimento.php (152)
in /var/www/fs_yii/protected/models/Estabelecimento.php (120)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (79)
2015/01/10 15:47:28 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22P02]: Invalid text representation: 7 ERROR:  invalid input syntax for integer: ""
LINE 1: ...enderecos(estabelecimento_id, endereco_id) values ('', '59')
                                                              ^. The SQL statement executed was: insert into yii_estabelecimento_enderecos(estabelecimento_id, endereco_id) values ('', '59')' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(152): CDbCommand->execute()
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(120): Estabelecimento->saveAssociativeTable('{{estabelecimen...', NULL, '59')
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(79): Estabelecimento->saveManyToMany()
#3 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(42): EstabelecimentoController->saveOrUpdate()
#4 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#5 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#9 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#10 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#11 /var/www/fs_yii/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 16:26:31 [error] [php] include(Email.php): failed to open stream: No such file or directory (/var/www/yii/framework/YiiBase.php:432)
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(395): spl_autoload_call()
#1 /var/www/yii/framework/db/ar/CActiveFinder.php(182): model()
#2 /var/www/yii/framework/db/ar/CActiveFinder.php(230): CActiveFinder->getModel()
#3 /var/www/yii/framework/db/ar/CActiveFinder.php(48): CActiveFinder->buildJoinTree()
#4 /var/www/yii/framework/db/ar/CActiveRecord.php(912): CActiveFinder->__construct()
#5 /var/www/yii/framework/db/ar/CActiveRecord.php(280): Estabelecimento->getActiveFinder()
#6 /var/www/yii/framework/db/ar/CActiveRecord.php(143): Estabelecimento->getRelated()
#7 /var/www/fs_yii/protected/models/Estabelecimento.php(112): Estabelecimento->__get()
#8 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(76): Estabelecimento->saveManyToMany()
#9 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(39): EstabelecimentoController->saveOrUpdate()
#10 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#11 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#12 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#13 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#14 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#15 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#16 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#17 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
in /var/www/fs_yii/protected/models/Estabelecimento.php (112)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (76)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (39)
2015/01/10 16:35:17 [error] [php] Invalid argument supplied for foreach() (/var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php:25)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#1 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): EstabelecimentoController->renderPartial()
#2 /var/www/yii/framework/web/CBaseController.php(126): require()
#3 /var/www/yii/framework/web/CBaseController.php(95): EstabelecimentoController->renderInternal()
#4 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#5 /var/www/yii/framework/web/CController.php(782): EstabelecimentoController->renderPartial()
#6 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(48): EstabelecimentoController->render()
#7 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#8 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#9 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#10 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#11 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#12 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#13 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#14 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
in /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php (25)
in /var/www/fs_yii/protected/views/estabelecimento/cadastro.php (3)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (48)
2015/01/10 19:32:55 [error] [exception.CException] exception 'CException' with message 'an Error occured while trying to remove data in associative table' in /var/www/fs_yii/protected/models/Estabelecimento.php:155
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(138): Estabelecimento->saveAssociativeTable('{{estabelecimen...', '196', '106')
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(112): Estabelecimento->saveRelations('enderecos', Array, Array)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(82): Estabelecimento->saveManyToMany()
#3 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(45): EstabelecimentoController->saveOrUpdate(Array)
#4 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#5 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#9 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#10 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#11 /var/www/fs_yii/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 19:33:51 [error] [exception.CException] exception 'CException' with message 'an Error occured while trying to remove data in associative table' in /var/www/fs_yii/protected/models/Estabelecimento.php:155
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(138): Estabelecimento->saveAssociativeTable('{{estabelecimen...', '197', '107')
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(112): Estabelecimento->saveRelations('enderecos', Array, Array)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(82): Estabelecimento->saveManyToMany()
#3 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(45): EstabelecimentoController->saveOrUpdate(Array)
#4 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#5 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#9 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#10 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#11 /var/www/fs_yii/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 19:34:22 [error] [exception.CException] exception 'CException' with message 'an Error occured while trying to remove data in associative table' in /var/www/fs_yii/protected/models/Estabelecimento.php:155
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(138): Estabelecimento->saveAssociativeTable('{{estabelecimen...', '198', '108')
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(112): Estabelecimento->saveRelations('enderecos', Array, Array)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(82): Estabelecimento->saveManyToMany()
#3 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(45): EstabelecimentoController->saveOrUpdate(Array)
#4 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#5 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#9 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#10 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#11 /var/www/fs_yii/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 19:36:12 [error] [exception.CException] exception 'CException' with message 'an Error occured while trying to remove data in associative table' in /var/www/fs_yii/protected/models/Estabelecimento.php:155
Stack trace:
#0 /var/www/fs_yii/protected/models/Estabelecimento.php(138): Estabelecimento->saveAssociativeTable('{{estabelecimen...', '199', '109')
#1 /var/www/fs_yii/protected/models/Estabelecimento.php(112): Estabelecimento->saveRelations('enderecos', Array, Array)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(82): Estabelecimento->saveManyToMany()
#3 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(45): EstabelecimentoController->saveOrUpdate(Array)
#4 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#5 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('cadastro')
#9 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#10 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#11 /var/www/fs_yii/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/cadastro
---
2015/01/10 19:45:17 [error] [php] Invalid argument supplied for foreach() (/var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php:37)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#1 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): EstabelecimentoController->renderPartial()
#2 /var/www/yii/framework/web/CBaseController.php(126): require()
#3 /var/www/yii/framework/web/CBaseController.php(95): EstabelecimentoController->renderInternal()
#4 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#5 /var/www/yii/framework/web/CController.php(782): EstabelecimentoController->renderPartial()
#6 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(44): EstabelecimentoController->render()
#7 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#8 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#9 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#10 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#11 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#12 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#13 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#14 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
in /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php (37)
in /var/www/fs_yii/protected/views/estabelecimento/cadastro.php (3)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (44)
2015/01/10 19:45:18 [error] [php] Invalid argument supplied for foreach() (/var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php:37)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#1 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): EstabelecimentoController->renderPartial()
#2 /var/www/yii/framework/web/CBaseController.php(126): require()
#3 /var/www/yii/framework/web/CBaseController.php(95): EstabelecimentoController->renderInternal()
#4 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#5 /var/www/yii/framework/web/CController.php(782): EstabelecimentoController->renderPartial()
#6 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(44): EstabelecimentoController->render()
#7 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#8 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#9 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#10 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#11 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#12 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#13 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#14 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
in /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php (37)
in /var/www/fs_yii/protected/views/estabelecimento/cadastro.php (3)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (44)
2015/01/10 19:45:37 [error] [php] Invalid argument supplied for foreach() (/var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php:37)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#1 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): EstabelecimentoController->renderPartial()
#2 /var/www/yii/framework/web/CBaseController.php(126): require()
#3 /var/www/yii/framework/web/CBaseController.php(95): EstabelecimentoController->renderInternal()
#4 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#5 /var/www/yii/framework/web/CController.php(782): EstabelecimentoController->renderPartial()
#6 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(44): EstabelecimentoController->render()
#7 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#8 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#9 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#10 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#11 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#12 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#13 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#14 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
in /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php (37)
in /var/www/fs_yii/protected/views/estabelecimento/cadastro.php (3)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (44)
2015/01/10 19:46:01 [error] [php] Invalid argument supplied for foreach() (/var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php:37)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#1 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): EstabelecimentoController->renderPartial()
#2 /var/www/yii/framework/web/CBaseController.php(126): require()
#3 /var/www/yii/framework/web/CBaseController.php(95): EstabelecimentoController->renderInternal()
#4 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#5 /var/www/yii/framework/web/CController.php(782): EstabelecimentoController->renderPartial()
#6 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(44): EstabelecimentoController->render()
#7 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#8 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#9 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#10 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#11 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#12 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#13 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#14 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
in /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php (37)
in /var/www/fs_yii/protected/views/estabelecimento/cadastro.php (3)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (44)
2015/01/10 19:47:11 [error] [php] Invalid argument supplied for foreach() (/var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php:36)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#1 /var/www/fs_yii/protected/views/estabelecimento/cadastro.php(3): EstabelecimentoController->renderPartial()
#2 /var/www/yii/framework/web/CBaseController.php(126): require()
#3 /var/www/yii/framework/web/CBaseController.php(95): EstabelecimentoController->renderInternal()
#4 /var/www/yii/framework/web/CController.php(869): EstabelecimentoController->renderFile()
#5 /var/www/yii/framework/web/CController.php(782): EstabelecimentoController->renderPartial()
#6 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(44): EstabelecimentoController->render()
#7 /var/www/yii/framework/web/actions/CInlineAction.php(49): EstabelecimentoController->actionCadastro()
#8 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#9 /var/www/yii/framework/web/CController.php(286): EstabelecimentoController->runAction()
#10 /var/www/yii/framework/web/CController.php(265): EstabelecimentoController->runActionWithFilters()
#11 /var/www/yii/framework/web/CWebApplication.php(282): EstabelecimentoController->run()
#12 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#13 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#14 /var/www/fs_yii/index.php(13): CWebApplication->run()
REQUEST_URI=/fs_yii/index.php/estabelecimento/cadastro
in /var/www/fs_yii/protected/views/estabelecimento/_form_save_or_update.php (36)
in /var/www/fs_yii/protected/views/estabelecimento/cadastro.php (3)
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (44)
2015/01/10 21:59:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "c".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('c')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('c')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/c
---
2015/01/10 22:04:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:05:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:05:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:05:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:05:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:05:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:05:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:05:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:05:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:05:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:05:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:13:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#3 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /var/www/fs_yii/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:18:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php:52
Stack trace:
#0 [internal function]: EstabelecimentoController->actionEditar('210')
#1 /var/www/yii/framework/web/actions/CAction.php(109): ReflectionMethod->invokeArgs(Object(EstabelecimentoController), Array)
#2 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(EstabelecimentoController), Object(ReflectionMethod), Array)
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/210
---
2015/01/10 22:18:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php:52
Stack trace:
#0 [internal function]: EstabelecimentoController->actionEditar('210')
#1 /var/www/yii/framework/web/actions/CAction.php(109): ReflectionMethod->invokeArgs(Object(EstabelecimentoController), Array)
#2 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(EstabelecimentoController), Object(ReflectionMethod), Array)
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/210
---
2015/01/10 22:18:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php:52
Stack trace:
#0 [internal function]: EstabelecimentoController->actionEditar('210')
#1 /var/www/yii/framework/web/actions/CAction.php(109): ReflectionMethod->invokeArgs(Object(EstabelecimentoController), Array)
#2 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(EstabelecimentoController), Object(ReflectionMethod), Array)
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:18:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page dofes not exist.' in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php:52
Stack trace:
#0 [internal function]: EstabelecimentoController->actionEditar('210')
#1 /var/www/yii/framework/web/actions/CAction.php(109): ReflectionMethod->invokeArgs(Object(EstabelecimentoController), Array)
#2 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(EstabelecimentoController), Object(ReflectionMethod), Array)
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('editar')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#8 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#9 /var/www/fs_yii/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/editar/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/
---
2015/01/10 22:20:48 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23503]: Foreign key violation: 7 ERROR:  update or delete on table "yii_estabelecimentos" violates foreign key constraint "estabelecimento_id_fk" on table "yii_estabelecimento_enderecos"
DETAIL:  Key (id)=(210) is still referenced from table "yii_estabelecimento_enderecos".. The SQL statement executed was: DELETE FROM "yii_estabelecimentos" WHERE "yii_estabelecimentos"."id"=210.
in /var/www/fs_yii/protected/controllers/EstabelecimentoController.php (73)
in /var/www/fs_yii/index.php (13)
2015/01/10 22:20:48 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23503]: Foreign key violation: 7 ERROR:  update or delete on table "yii_estabelecimentos" violates foreign key constraint "estabelecimento_id_fk" on table "yii_estabelecimento_enderecos"
DETAIL:  Key (id)=(210) is still referenced from table "yii_estabelecimento_enderecos".. The SQL statement executed was: DELETE FROM "yii_estabelecimentos" WHERE "yii_estabelecimentos"."id"=210' in /var/www/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(1807): CDbCommand->execute()
#1 /var/www/yii/framework/db/ar/CActiveRecord.php(1227): CActiveRecord->deleteByPk(210)
#2 /var/www/fs_yii/protected/controllers/EstabelecimentoController.php(73): CActiveRecord->delete()
#3 [internal function]: EstabelecimentoController->actionDelete('210')
#4 /var/www/yii/framework/web/actions/CAction.php(109): ReflectionMethod->invokeArgs(Object(EstabelecimentoController), Array)
#5 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(EstabelecimentoController), Object(ReflectionMethod), Array)
#6 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('delete')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('estabelecimento...')
#11 /var/www/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#12 /var/www/fs_yii/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/fs_yii/index.php/estabelecimento/delete/210
HTTP_REFERER=http://localhost/fs_yii/index.php/estabelecimento/index
---
